packages/jest-circus/src/legacy-code-todo-rewrite/jestAdapter.ts

```diff
@@ -12,7 +12,8 @@ import type {TestResult} from '@jest/test-result';
 import type {RuntimeType as Runtime} from 'jest-runtime';
 import type {SnapshotStateType} from 'jest-snapshot';
 
-const FRAMEWORK_INITIALIZER = require.resolve('./jestAdapterInit');
+const FRAMEWORK_INITIALIZER = path.resolve(__dirname, './jestAdapterInit');
+const EXPECT_INITIALIZER = path.resolve(__dirname, './jestExpect.js');
```

Comment:
changes not related, just some local cleanup from looking into not adding the globals ti the global env. Went with a simpler solution, but I liked these refactorings 🤷